tests/system/CodeIgniterTest.php

```diff
@@ -622,4 +623,83 @@ public function testPageCacheSendSecureHeaders()
         stream_filter_remove($outputStreamFilter);
         stream_filter_remove($errorStreamFilter);
     }
+
+    /**
+     * @param mixed $cacheQueryStringValue
+     * @param int   $expectedPagesInCache
+     * @param array $testingUrls
```

Comment:
If you use a single type, use parameter type declaration instead of doc comment.